modules/v2x/fusion/configs/test/ft_config_manager_test.cc

```diff
@@ -0,0 +1,50 @@
+/***************************************************************************
+ *
+ * Copyright (c) 2018 Baidu.com, Inc. All Rights Reserved
+ *
+ **************************************************************************/
+
+/**
+ * File: km_test.cc
+ * Brief: The km test program for the v2x module.
+ * Author: Jary (lizheng23@baidu.com).
+ */
+#include "configs/ft_config_manager.h"
+
+#include "gtest/gtest.h"
+
+namespace v2x {
+namespace ft {
+
+// using namespace v2x::ft::track;
+
+// using base::SensorType;
+
+TEST(FTConfigManager, read_and_write) {
+  FTConfigManager *ft_config_manager_ptr = FTConfigManager::Instance();
+  auto& fusion_params = ft_config_manager_ptr->fusion_params_.params;
+  EXPECT_TRUE(fusion_params.score_params().check_type());
+  EXPECT_EQ(fusion_params.score_params().prob_scale(), 0.125);
+  // fusion_params.mutable_score_params()->set_prob_scale(0.3);
+  
+}
+
+TEST(FTConfigTools, readfile) {
+  fusion::FusionParams params;
+  EXPECT_TRUE(tools::ParseProtobufFromFile(std::string("./test_data/fusion_params.pt"), &params));
+  EXPECT_FALSE(tools::ParseProtobufFromFile(std::string("./test_data/fusion_params_wrong.pt"), &params));
+}
+}  // namespace ft
+}  // namespace v2x
+
+int main(int argc, char** argv) {
+  testing::InitGoogleTest(&argc, argv);
```

Comment:
Avoid manually configured argc/argv. Use facilities Bazel test provides for such task.